### DISCUSSION TITLE
Declaração de imunidade/isenção de recolhimento do IRRF/CSRF.

### DISCUSSION THREAD

**Discussion by PySulBR** [apyb/comunidade#134] [Pergunta]
Olá,
uma empresa pediu a declaração de imunidade/isenção de recolhimento do IRRF/CSRF da APyB.

@cecivieira

**Comment by cecivieira** [comment]
Fiz essa pergunta para a nossa contadora. Assim que soubermos a resopsta, informarei.

**Comment by cecivieira** [comment]
@PySulBR amanhã a contadora enviará um modelod e declaração. Então, espero que até o final do dia nós te enviemos.

**Comment by PySulBR** [comment]
olá @cecivieira, algum retorno?

**Reply by cecivieira** [reply]
https://drive.google.com/file/d/19ZzE2FShvLVHEOUsuMgcHBYfM-zH7MIb/view?usp=sharing

Desculpa a demora

**Reply by PySulBR** [reply]
Obrigadão <3

**Comment by cecivieira** [comment]
[Emilly-horta](https://github.com/Emilly-horta) verifica se essa issue já atende a sua dúvida

**Reply by emillyhorta** [reply]
Posso utilizar essa declaração ?

**Reply by cecivieira** [reply]
Sim
